modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -82,7 +82,7 @@ To configure a domain for the first time:
 ** `api-proxy-listener-http`: The listener configuration for HTTP communications 
 ** `api-proxy-listener-https`: The listener configuration for HTTPS communications
 +
-. Copy the Maven artifact to the `../domains/` folder of your running runtime distribution, for example, `gateway-proxy-domain-1.0.0-mule-application-template.jar`.
+. Copy the Maven artifact to the `domains` folder of your running Mule distribution, for example, `gateway-proxy-domain-1.0.0-mule-application-template.jar` and replace `mule-application-template` with `mule-domain`, resulting in the string: `gateway-proxy-domain-1.0.0-mule-domain.jar`.
```

Comment:
I thought of it too, but then let it go since it was harder to link the two when trying to understand what needs to be done. I made into two steps now anyway.